contracts/interchain-token-service/tests/flow_limit.rs

```diff
@@ -339,91 +339,79 @@ fn add_flow_out_fails_on_exceeding_flow_limit() {
     );
 }
 
-const LARGE_FLOW_LIMIT: i128 = i128::MAX - 50;
-const OVERFLOW_CASES: &[(i128, i128)] = &[
-    (LARGE_FLOW_LIMIT - 1, 2),
-    (i128::MAX - 100, 2),
-    (LARGE_FLOW_LIMIT, 1),
-];
+#[test]
+fn add_flow_fails_on_flow_comparison_overflow() {
+    let flow_limit = i128::MAX - 50;
+    let cases = std::vec![(flow_limit - 1, 2), (i128::MAX - 100, 2), (flow_limit, 1),];
 
-enum OverflowTestDirection {
-    InThenOut,
-    OutThenIn,
-}
+    for case in &cases {
```

Comment:
```suggestion
    for (flow_in, flow_out) in &cases {
```
destructure here to avoid the need for `.0` and `.1`